lib/state/src/index.ts

```diff
@@ -1,6 +1,6 @@
 import * as fs from 'fs'
 const target = process.env.INIT_CWD || process.cwd()
-const stateFile = target ? `${target}/.toolkitstate.json` : '.toolkitstate.json'
+const stateDir = target ? `${target}/.toolkitstate` : '/.toolkitstate'
```

Comment:
**suggestion**: this (and constructing the stage `.json` paths below) should use `path.join` instead of a template string, currently this probably won't work on Windows

**issue**: i don't think the leading `/` on the fallback here is correct, that would try to create the directory at the root of the filesystem (which will almost certainly fail)